quizyapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.1.6 on 2021-02-25 23:40
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Answer',
+            fields=[
+                ('id', models.CharField(max_length=32, primary_key=True, serialize=False, unique=True)),
+                ('text', models.TextField()),
+                ('is_correct', models.BooleanField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('id', models.IntegerField(primary_key=True, serialize=False, unique=True)),
+                ('name', models.TextField(unique=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Question',
+            fields=[
+                ('id', models.CharField(max_length=32, primary_key=True, serialize=False, unique=True)),
+                ('text', models.TextField()),
+                ('difficulty', models.TextField(choices=[('easy', 'Easy'), ('medium', 'Medium'), ('hard', 'Hard')])),
+                ('answers', models.ManyToManyField(to='quizyapp.Answer')),
+                ('category', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='quizyapp.category')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='UserPoints',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('points', models.IntegerField()),
+                ('user', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL)),
```

Comment:
Może jednak to powinna być relacja 1:1?

Reply:
z tego modelu w ogóle zrezygnowałem na rzecz UserAnswer, UserPoints usunę